integration_test/it-setup.sh

```diff
@@ -157,7 +157,8 @@ start_containers() {
 	echo "- Running ${KDB_CONTAINER} ..."
 	# DataBroker container options
 	rc=0
-	docker run -d ${DOCKER_OPT} ${KDB_DOCKER_OPT} "${KDB_IMAGE}" || rc=1
+	echo "Starting run -d ${DOCKER_OPT} ${KDB_DOCKER_OPT} ${KDB_IMAGE} ${KDB_OPT}"
```

Comment:
```echo "\$ docker run -d ${DOCKER_OPT} ${KDB_DOCKER_OPT} ${KDB_IMAGE} ${KDB_OPT}"```

Reply:
This can actually be removed, I just added it during troubleshooting.